lib/usr/app/jcfg/jcfg_private.h

```diff
@@ -90,6 +90,15 @@ jcfg_get_json_token(struct jcfg *cfg)
     return (cfg) ? cfg->tok : NULL;
 }
 
+/**
+ * Free the internal umem structure.
+ * @internal
+ *
+ * @param hdr
+ *   The generic header structure pointer.
+ */
+void jcfg__umem_free(jcfg_hdr_t *hdr);
```

Comment:
Why use the double underscore here? Is it becuase its private? I don't think its necessary, and is not the convention used anywhere else.